index.html

```diff
@@ -3451,6 +3451,25 @@ <h1>DID Methods</h1>
               <a href="https://gitlab.com/proximax-enterprise/siriusid/sirius-id-specs/-/blob/master/docs/did-method-spec.md">ProximaX SiriusID DID Method</a>
           </td>
         </tr>
+
```

Comment:
In anticipation of a previous [cleanups PR](https://github.com/w3c/did-spec-registries/pull/269) being merged, it would be nice to change this to two space indents, and drop the trailing blank line.

Reply:
And... it was merged.  Please update the indents and remove the leading/trailing blank lines to match the common style.